sw/airborne/firmwares/rotorcraft/autopilot_static.c

```diff
@@ -173,8 +173,14 @@ void autopilot_static_periodic(void)
     SetCommands(commands_failsafe);
   } else {
     guidance_v_run(autopilot_in_flight);
-    guidance_h_run(autopilot_in_flight);
+    guidance_h_run(autopilot_in_flight);    
     SetRotorcraftCommands(stabilization_cmd, autopilot_in_flight, autopilot_motors_on);
+
+    if (lock_wings == 1 && autopilot_motors_on && commands[COMMAND_THRUST] > 0) {
```

Comment:
Didn't test this yet (Friday)... but something like this I suppose?

I tried to include modules.h in autopilot_utils directly, but that gives all sorts of trouble :(